package.json

```
@@ -11,20 +11,20 @@
     "dist/*.d.ts"
   ],
   "scripts": {
-    "serve-static": "live-server --open=demo --watch=dist/o_spreadsheet.js,dist/o_spreadsheet.xml,demo",
+    "serve-static": "live-server --open=demo --watch=build/o_spreadsheet.js,build/o_spreadsheet.xml,demo",
     "dev": "npm-run-all build --parallel server serve-static watch:*",
     "server": "node tools/server/main.js",
     "build:js": "tsc --module es6 --incremental",
     "bundle:js": "rollup -c -m dev",
     "bundle:xml": "node tools/bundle_xml/main.js",
-    "build": "npm-run-all build:js bundle:*",
+    "build": "npm-run-all build:js bundle:js 'bundle:xml -- -outDir build'",
```

multi-characters args are usually prefixed with `--` while single character are prefixed with `-`
```suggestion
    "build": "npm-run-all build:js bundle:js 'bundle:xml -- --outDir build'",
```